packages/dayjs/src/dayjs-utils.ts

```diff
@@ -102,7 +102,7 @@ export default class DayjsUtils implements IUtils<defaultDayjs.Dayjs> {
       return null;
     }
 
-    return this.dayjs(value, format);
+    return this.dayjs(value, format, this.locale);
```

Comment:
Are you sure `this.locale` is always defined? Will it crash if locale is not defined?